src/plugins/validation.ts

```diff
@@ -132,12 +130,59 @@ const isBetween = (
  * @returns {function(*): (boolean|string)}
  */
 const isDateValid = (
-  message?: string
+    message?: string
 ): ((v: string) => string | true | undefined) => {
   message = message || `Invalid Date`;
   // validate date isn't something like 12/DD/YYYY
   return (v: string) => {
-    return (/^[0-9]*$/.test(v.replaceAll(/\//g, ""))) || message 
+    return (/^[0-9]*$/.test(v.replaceAll(/\//g, ""))) || message
+  };
+};
+
+/**
+ * Validator that validates if input is a valid Email
+ * Returns the error message otherwise.
+ *
+ * @param (string) email as "test@mail.mil"
+ * @returns {function(*): (boolean|string)}
+ */
+const isEmail = (): ((v: string) => string | true | undefined) => {
+  return (v: string) => {
+    if (v !== "") {
+      if (/[a-z0-9]+@[a-z]+\.[a-z]{3}/.test(v)) {
+        return "Please use standard domain format, like ‘@mail.mil’"
+      } else if (v.indexOf("@") < 0) {
+        return "Please include an '@’ symbol in the email address."
+      } else if (/[a-z0-9]+@[a-z]+\.[ gov,mil ]/.test(v)) {
+        return "Please use your .mil or .gov email address."
+      }
+    }
+    return true;

```

Comment:
validation isn't hitting some invalid entries -- these do not trigger an error 
`a@`, `foo@@@mail.mil` `.mil` `foo@mail` `foo @ mail` etc. 
Need to check only 1 `@` symbol, no spaces, etc. There's a regex in old atat that should be helpful. I don't believe it detected all invalid emails, but the more robust ones were failing SonarCloud I think... 
`private emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;`